### PR TITLE
Create PFDCMClient with mock data

### PR DESCRIPTION
**Pull request by matankb**
Let me know what you think about the structure of the API. `PFDCMFilters` can be expanded to include as many filters as needed.

### CODE REVIEW

File: src/api/pfdcm/client.ts [hunk 1]
```diff
@@ -1 +1,70 @@
...
+    return parseRawDcmData(mockData);
+  }
+
+  static async queryByStudy(study: string, filters: PFDCMFilters) {
+    return parseRawDcmData(mockData);
+  }
+  
+}
+
+export default PFDCMClient;
```
Comment (zrthxn) [inline src/api/pfdcm/client.ts]: Maybe put this client in the `api/pfdcmclient.ts` file? 
Makes it easier to import with just `import Client from 'api/pfdcmclient'`

**Review by zrthxn — COMMENTED** [review]
Some of the types like date, studyDate, modality, patientSex, status etc. could be more specific. Like instead of just being the `string` type maybe some can be a union of literals? For eg it can be something like `patientSex: "A" | "B" | "C" | string`. Basically have more discrete options where possible instead of just `string`.

For something like date, it'd probably be good to also have a Date type, and the functions would convert the incoming string to a JS Date object?

File: src/api/pfdcm/client.ts (hunk 1 above)
Comment (matankb) [reply src/api/pfdcm/client.ts]: I renamed it to `index.ts` inside the folder instead, so it can be imported the same way.

Comment (zrthxn) [reply src/api/pfdcm/client.ts]: Alright 👍

**Comment by zrthxn** [comment]
@matankb is this ready to be merged for now to be used as a dummy api?

**Comment by matankb** [comment]
@zrthxn If it looks good to you, yeah.

**Pull request merged**